frappe/commands/utils.py

```diff
@@ -342,7 +342,7 @@ def mariadb(context):
 		'-p'+frappe.conf.db_password,
 		frappe.conf.db_name,
 		'-h', frappe.conf.db_host or "localhost",
-		'--pager=less -SFX',
+		'--pager=less -SFX','--safe-updates',
```

Comment:
Add a newline here

Reply:
Thanks for the suggestion, added a newline and committed again.